tests/compare_result.py

```diff
@@ -23,7 +23,8 @@ def read_energyresult_dat(filename):
     data_energy = data_raw[:, :N_energies]
 
     n_data = (data_raw.shape[1] - N_energies) // 2
-    assert n_data % 2 == 0
+    if cmplx:
+        assert n_data % 2 == 0 , "found odd n_data={} (N_energies={})".format(n_data, N_energies)
```

Comment:
Maybe this `n_data` test can be just removed, because the data shape will be checked against the reference data anyway.

Then, we could also remove the `cmplx` parameter.

Reply:
done, removed the cmplx parameter